schemas/bedrock-kms-http.js

```diff
@@ -0,0 +1,192 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const controller = {
+  title: 'controller',
+  type: 'string'
+};
+
+const referenceId = {
+  title: 'referenceId',
+  type: 'string'
+};
+
+const delegator = {
+  anyOf: [{
+    type: 'string'
+  }, {
+    type: 'array',
+    minItems: 1,
+    items: {type: 'string'}
+  }]
+};
+const invoker = {
```

Comment:
Add whitespace for inter-file consistency.

```suggestion

const invoker = {
```

Reply:
addressed here 44d1308